tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -442,7 +449,29 @@
             </actions>
         </data>
         
-        <data format="bam" name="output" label="${tool.name} on ${on_string}: aligned reads (sorted BAM)">
+        <data format="bam" name="output_1" label="${tool.name} on ${on_string}: aligned reads (sorted BAM)">
```

Comment:
@jmchilton good point.   You could probably name the sam output as "output_sam".